recipes/ogdf/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set name = "ogdf" %}
+{% set version = "2022.02" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/ogdf/ogdf/archive/refs/tags/dogwood-202202.tar.gz
+  sha256: 308cc2749c6a63520f7979bac86a04066dfea2fb9d3a5e89831318db404bfbf5
+
+  patches:
+    - cmake.patch
+
+build:
+  number: 0
+  skip: True  # [win]
+  run_exports:
+    - {{ pin_subpackage('ogdf', max_pin='2022.02') }}
+
+requirements:
+  build:
+    - {{ compiler("cxx") }}
+    - backward-cpp                           # [build_platform != target_platform]
+    - pugixml                                # [build_platform != target_platform]
+    - coincbc                                # [build_platform != target_platform]
+    - libdwarf-dev                           # [build_platform != target_platform]
+  host:
+    - cmake >=3.1
+    - make               # [not win]
+    - ninja              # [win]
+    - backward-cpp
+    - pugixml
+    - coincbc
+    - libdwarf-dev
+  run:
+    # To prevent DSO linking errors
+    - coin-or-clp
+    - coin-or-utils
+
+test:
+  commands:
+    - test -f $PREFIX/lib/libOGDF.so                 # [linux]
+    - test -f $PREFIX/lib/libOGDF.dylib              # [osx]
+    - if not exist %LIBRARY_LIB%\\OGDF.lib exit 1    # [win]
```

Comment:
Please add an existence check for at least one file of each of the following categories: Cmake Config, Header,